src/modules/newScores.js

```diff
@@ -0,0 +1,27 @@
+const name = document.querySelector('#name');
+const score = document.querySelector('#scores');
+
+const newScore = async () => {
+  try {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:OajjUViqUBOUtMbxRx8L/scores/', {
+      method: 'POST',
+      body: JSON.stringify({
+        user: name.value,
+        score: score.value,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    // eslint-disable-next-line no-unused-vars
+    const data = await response.json();
+
+    name.value = '';
+    score.value = '';
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
```

Comment:
- [ ] for **_[Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)_** kindly remove the `console.log` from your repo. It’s ugly, it kills performance and it can make confidential data visible to anyone using the browser tools to look at your website. please fix it and you will be good to go ^^